app/src/main/java/com/nytimes/android/external/register/buy/BuyFragment.kt

```diff
@@ -250,13 +250,10 @@ class BuyFragment : Fragment() {
         PermissionHandler.handlePermissionResult(requestCode, requireActivity(), *grantResults)
     }
 
-    override fun onDestroy() {
-        super.onDestroy()
-        if (requireActivity().isFinishing) {
-            val bundle = Bundle()
-            bundle.putInt(GoogleUtil.RESPONSE_CODE, GoogleUtil.RESULT_USER_CANCELED)
-            requireActivity().setResult(AppCompatActivity.RESULT_OK)
-        }
+    override fun backPressed() {
+        val bundle = Bundle()
+        bundle.putInt(GoogleUtil.RESPONSE_CODE, GoogleUtil.RESULT_USER_CANCELED)
+        requireActivity().setResult(AppCompatActivity.RESULT_OK)
```

Comment:
I'm not quite sure what this bundle is supposed to be doing, but it's been doing this for a bit now...

https://github.com/nytimes/Register/pull/80/files#diff-b0241344656d8865d308c2e40e8628dcL309

https://github.com/nytimes/Register/pull/70/files#diff-3789529ff441a35fd2abcd16bdb73bd4L329

I've not changed anything with it, but still confused

Reply:
Check out `StoreFrontGooglePurchaseActivity` in phoenix.  We listen for this in `onActivityResult`

Reply:
I see that we're looking for extras in that class.

The part that confuses me is where do we actually do anything with this bundle? Shouldn't it be set somewhere or actually passed to something? It looks like we just create a bundle, put data in it, and then don't pass that bundle anywhere. Is that intended?